### PR TITLE
Reservations updated and testet

### PR DESCRIPTION
**Pull request by SebastianQJ**

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java [hunk 1]
```diff
@@ -12,16 +12,15 @@
     private int id;
 
     @ManyToOne()
-    @JoinColumn(name = "showing.id", nullable = false)
+    @JoinColumn(name = "showing.id", nullable = true)
     private Showing showing;
     @Column(name = "seat_number")
     private int seat;
     @Column(name = "customers")
     private String phoneNumber;
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java [hunk 2]
```diff
@@ -12,16 +12,15 @@
...
 
 
-
-
+    public Reservation(){}
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Reservation/Models/Reservation.java]: remove spacing

**Review by AnderskdKnudsen — COMMENTED** [review]
Hoooooly moly. Near-mint pull request

**Comment by SebastianQJ** [comment]
Done

**Review by AnderskdKnudsen — APPROVED** [review]

**Pull request merged**